src/test/java/nextstep/courses/domian/SessionTest.java

```diff
@@ -0,0 +1,117 @@
+package nextstep.courses.domian;
+
+import nextstep.courses.CannotRecruitException;
+import nextstep.courses.domain.Amount;
+import nextstep.courses.domain.CoverImage;
+import nextstep.courses.domain.CoverImageFileName;
+import nextstep.courses.domain.CoverImagePixel;
+import nextstep.courses.domain.CoverImageSize;
+import nextstep.courses.domain.Session;
+import nextstep.courses.domain.SessionDuration;
+import nextstep.courses.domain.SessionEnrolment;
+import nextstep.courses.domain.SessionStatusType;
+import nextstep.courses.domain.SessionStudent;
+import nextstep.courses.domain.Students;
+import nextstep.users.domain.NsUser;
+import nextstep.users.domain.NsUserTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class SessionTest {
+
+    @Test
+    @DisplayName("무료 강의라면 인원수 제한없이 신청이 가능하다.")
+    void enrolment_무료() {
+        Students students = new Students(new ArrayList<>(List.of(NsUserTest.SANJIGI, NsUserTest.JAVAJIGI)));
+        SessionStudent sessionStudent = new SessionStudent(students);
+        Amount amount = new Amount();
+        NsUser newUser = new NsUser(3L, "test", "test", "test", "test");
+        SessionEnrolment sessionEnrolment = new SessionEnrolment(sessionStudent, SessionStatusType.RECRUITMENT, amount, true);
+        SessionDuration sessionDuration = new SessionDuration(LocalDateTime.now(), LocalDateTime.now().plusDays(3));
+        CoverImage coverImage = new CoverImage(new CoverImageFileName("test.png"), new CoverImageSize(300), new CoverImagePixel(300, 200));
+
+        Session session = new Session(1L, sessionDuration, sessionEnrolment, coverImage);
+        session.enrolment(newUser, 0L);
+
+        assertThat(students.totalCount()).isEqualTo(3);
```

Comment:
```suggestion
        Students students = new Students(new ArrayList<>(List.of(NsUserTest.SANJIGI, NsUserTest.JAVAJIGI)));

        int actual = students.totalCount();
        int expected = 2;
        
        assertThat(actual).isEqualTo(expected);
```

테스트  F.I.R.S.T 원칙을 한번 고민해보아요.
너무 복잡한것 같아요.
테스트하고자하는 본질이 무엇인지 고민해보아도 좋을것 같네요.

저는 개인적으로 테스트를 최대한 심플하게 바라보기 위해서,
`actual`,`expected`를 선언해놓고 시작하면 좋더라구요 😄

Reply:
좋은 팁인거 같습니다 👍  직관적으로 필요한 부분만 볼 수 있는 좋은 방법인거 같아요 ㅎㅎ! 바로 적용했습니다
https://github.com/next-step/java-lms/pull/238/commits/30c1f0c5e56c64440dd22b64eaa4f266fd1cd72c